docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,303 @@
+
+````@example creating_geometry
+land_path = GeoMakie.assetpath("ne_110m_land.geojson")
+````
+
+Read the land polygons into a `GeoJSON.FeatureCollection`.
+
+````@example creating_geometry
+land_geo = GeoJSON.read(read(land_path, String))
+````
+
+create a figure with a `GeoAxis` from GeoMakie, that can handle the projections between different CRSs.
+
+````@example creating_geometry
+fig = Figure(size=(1000, 500));
+ga = GeoAxis(
+    fig[1, 1];
+    source=crs, # `source` and `dest` set the CRS
+    dest=dest,
+    xticklabelsvisible = false,
+    yticklabelsvisible = false,
+);
+nothing #hide
+````
+
+Plot `land` for context.
+
+````@example creating_geometry
+poly!(ga, land_geo, color=:black)
+fig
+````
+
+Now let's plot a `Polygon` like before, but now on coordinate reference system (CRS) that is different from our data
+
+````@example creating_geometry
+plot!(multipolygon; color = :green)
+fig
+````
+
+OK, now what if we want to plot geometries on the same figure that are in a different Coordinate Reference System (CRS)?
+
+This is our new source `CRS`: the UTM (Universal Transverse Mercator) zone 10N [EPSG:32610](https://epsg.io/32610).  It's a type of [transverse Mercator projection](https://en.wikipedia.org/wiki/Transverse_Mercator) (a cylindrical projection) that covers the west coast of the U.S. and some regions in Canada.
+
+````@example creating_geometry
+crs2 = GFT.EPSG(32610)
+````
+
+Create a polygon in the new coordinate system (we're working in meters now, not lat/lon)
+````@example creating_geometry
+r = 1000000;
+ϴ = 0:0.01:2pi;
+x = r .* cos.(ϴ).^3 .+ 500000;
+y = r .* sin.(ϴ) .^ 3 .+5000000;
+````
+
+Now create a `LinearRing` from `Points`
+
+````@example creating_geometry
+ring3 = GI.LinearRing(Point.(zip(x, y)))
+````
+
+Now create a `Polygon` from the `LineRing` with new the `CRS`
+
+````@example creating_geometry
+polygon3 = GI.Polygon([ring3]; crs=crs2)
+````
+
+Now plot on existing GeoAxis. 
+
+!!! note
+    The keyword argument `source` is used to specify the source `CRS` of that particular plot, when plotting on an existing `GeoAxis`.
+
+````@example creating_geometry
+plot!(ga,polygon3; color=:green, source=crs2)
+fig
+````
+
+Great, we can make geometries and plot them on a map... now let's export the data to common geospatial data formats.
+
+Typically, you'll also want to include attibutes with your geometries. Attibutes are simply data that is attibuted to each geometry. The easiest way to do this is to create a table with a `:geometry` column. Let's do this using [`DataFrames`](https://github.com/JuliaData/DataFrames.jl).
+
+````@example creating_geometry
+using DataFrames
+import Shapefile
+import GeoJSON
+import GeoParquet
+
+
+df = DataFrame(geometry=[polygon1, polygon2])
+````
+
+Now let's add a couple of attributes to the geometries
+
+````@example creating_geometry
+df[!,:id] = ["a", "b"]
```

Comment:
What do the `!` do?

Reply:
That's DataFrames.jl syntax to mutate a column.  Pretty standard over there, we can link to a description though.